packages/bot/src/stores/summary-store.js

```diff
@@ -1,21 +1,24 @@
 import {
     observable,
-    action }          from 'mobx';
-import { observer }   from '../utils/observer';
+    action,
+}                    from 'mobx';
+import { observer }  from '../utils/observer';
 
 export default class SummaryStore {
-    @observable currency        = '';
-    @observable lost_contracts  = 0;
-    @observable number_of_runs  = 0;
-    @observable total_profit    = 0;
-    @observable total_payout    = 0;
-    @observable total_stake     = 0;
-    @observable won_contracts   = 0;
-    
+    @observable currency = '';
+    @observable summary = {
```

Comment:
Could we rename this to `statistics`? I think it more accurately explains the contents of the object, and we can later reference it more clearly using `statistics` rather than `s`.

Reply:
@aaron-binary no i think its better since the component and store name and everything is summary.